### ISSUE TITLE
{Freeway-Bug}[Adaptive Card > Image Gallary.Json]: Images under cards are not accessible through the swipe gesture.

### ISSUE THREAD

**Issue by vagpt** [microsoft/Teams-AdaptiveCards-Mobile#175]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "Image Gallary.json" control and double tap to activate it.
7. Perform right swipe and observe the issue.

**Actual Result:**
While navigating under the 'Image Gallery' card, Images under cards are not accessible through the swipe gesture.

**Expected Result:**
Images should be accessible through the swipe gesture when user navigates the images gallery.

**Note:**
Same issue is observed for the images under FoodOrder.Json.
Same issue is repro throughout the application for the images.

**User Experience:**
It will impact the screen reader user as they will not be able to navigate to the images and also do not perform the desirable action what they want to do on the card.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/3a412383-09bd-45b7-8399-fa093db3f128

### Card JSON

```JSON
NA
```

**Comment by MengliElmendorf137** [comment]
The issue is not reproducible. Attached the screen recording for reference.
https://github.com/user-attachments/assets/316ddebd-c7b4-4e83-9ec0-7ccb37b17ebd

**Comment by vagpt** [comment]
This issue is still repro on the build 1.0 (2.4.0-beta.24.11.3.1) so we will reactivate this issue in next month.

**Comment by vagpt** [comment]
Attachment is added for the reference

https://github.com/user-attachments/assets/656ce2ee-3f97-4221-b6b2-a41da2ffaddf

**Comment by vagpt** [comment]
Hi @MengliElmendorf137 and @GabrielMedAlv,

Could you please provide the ETA to fix this issue? Due to this bug the product is getting Non-Compliant.

**Version:**
Version 1.0 (2.4.0-beta.25.1.26.1)

**Comment by GabrielMedAlv** [comment]
Hi @vagpt . I tested on latest main changes on 2 devices and simulator, and is working as expected. Will notify when new version is released to test it again.
